lib/bolt/cli.rb

```diff
@@ -515,6 +523,37 @@ def list_tasks
       raise Bolt::CLIError, "Failure while reading task metadata"
     end
 
+    def get_task_info(task_name)
+      task_info = {}
+      # TODO: should this formatting be done somewhere else? I think it makes
```

Comment:
Seems ok.

Reply:
Long term maybe in puppet strings. I think it makes sense to generate the usage line in the human outputter and not display it with json output for now.

Reply:
Should the json output just display hash returned from `compiler.task_signature(task_name).task_hash` ? Or should I edit that at all?